src/test/ceph_objectstore_tool.py

```diff
@@ -44,7 +44,8 @@ def wait_for_health():
     print "Wait for health_ok...",
     tries = 0
     while call("./ceph health 2> /dev/null | grep -v 'HEALTH_OK\|HEALTH_WARN' > /dev/null", shell=True) == 0:
-        if ++tries == 150:
+        tries = tries + 1
```

Comment:
could be
```py
tries += 1
```

and my first name is "Kefu"

also ideally, you could have an empty line before the "Signed-off-by" line. "git commit -s" can do it nicely.